be/src/formats/parquet/file_reader.cpp

```diff
@@ -382,7 +384,7 @@ bool FileReader::_is_integer_type(const tparquet::Type::type& type) {
 void FileReader::_prepare_read_columns() {
     const vectorized::HdfsScannerContext& param = *_scanner_ctx;
     for (auto& materialized_column : param.materialized_columns) {
-        int field_index = _file_metadata->schema().get_column_index(materialized_column.col_name, param.case_sensitive);
+        int field_index = _file_metadata->schema().get_column_index(materialized_column.col_name);
```

Comment:
Is `materialized_column.col_name` always lower case?

Reply:
Not true. Yes. that's why some test cases have failed. I'll fix it too.